src/Template/Pages/home.ctp

```diff
@@ -42,71 +42,71 @@ $cakeDescription = 'CakePHP: the rapid development php framework';
     <header>
         <div class="header-image">
             <?= $this->Html->image('http://cakephp.org/img/cake-logo.png') ?>
-            <h1>Get the Ovens Ready</h1>
+            <h1><?= __d('cake', 'Get the Ovens Ready') ?></h1>
         </div>
     </header>
     <div id="content">
         <div class="row">
             <div class="row">
                 <div class="columns large-12 ctp-warning checks">
-                    <p>Please be aware that this page will not be shown if you turn off debug mode unless you disable the NotFoundException in src/Template/Pages/home.ctp.</p>
+                    <p><?= __d('cake', 'Please be aware that this page will not be shown if you turn off debug mode unless you disable the NotFoundException in src/Template/Pages/home.ctp.') ?></p>
                 </div>
             </div>
             <?php Debugger::checkSecurityKeys(); ?>
             <div id="url-rewriting-warning" class="columns large-12 url-rewriting checks">
-                <p class="problem">URL rewriting is not properly configured on your server.</p>
+                <p class="problem"><?= __d('cake', 'URL rewriting is not properly configured on your server.') ?></p>
                 <p>
-                    1) <a target="_blank" href="http://book.cakephp.org/3.0/en/installation.html#url-rewriting">Help me configure it</a>
+                    1) <a target="_blank" href="http://book.cakephp.org/3.0/en/installation.html#url-rewriting"><?= __d('cake', 'Help me configure it') ?></a>
                 </p>
                 <p>
-                    2) <a target="_blank" href="http://book.cakephp.org/3.0/en/development/configuration.html#general-configuration">I don't / can't use URL rewriting</a>
+                    2) <a target="_blank" href="http://book.cakephp.org/3.0/en/development/configuration.html#general-configuration"><?= __d('cake', "I don't / can't use URL rewriting") ?></a>
```

Comment:
You would need to make all urls {n} parts as they are not subject of translation, ever.